osrf/vrx
# Scoring plugin part2 - collisions

This pull request adds collision tracking to the scoring plugin.

### How to test it?

Run the simulation:
```
ros2 launch vrx_ros competition_local.launch.py ign_args:="-v 4 -r sydney_regatta.sdf
```

Load the WAM-V:
```
ros2 launch vrx_ign spawn.launch.py name:=wamv world:=sydney_regatta model:=wam-v x:=-532 y:=162 z:=0 R:=0 P:=0 Y:=1
```

Run the joystick launch file:
```
ros2 launch vrx_ign usv_joy_teleop.py
```

Echo the `/vrx/task/info` topic:
```
ign topic -e -t /vrx/task/info
```

Now, watch the `num_collisions` field while you teleoperate the WAM-V. Hit a few buoys and see how the `num_collisions` increment. Note that there's a `3` second buffer between hits.
